cmake/schemes/url_sha1_odb-sqlite_autotools.cmake.in

```diff
@@ -0,0 +1,81 @@
+# Copyright (c) 2013, 2015 Ruslan Baratov, Alexandre Pretyman
+# All rights reserved.
+
+cmake_minimum_required(VERSION 3.0)
+project(Hunter)
+
+include(ExternalProject) # ExternalProject_Add
+
+list(APPEND CMAKE_MODULE_PATH "@HUNTER_SELF@/cmake/modules")
+
+include(hunter_fatal_error)
+include(hunter_status_debug)
+include(hunter_test_string_not_empty)
+include(hunter_autotools_configure_options)
+
+hunter_status_debug("Scheme: url_sha1_odb-sqlite_autotools")
+
+set(HUNTER_STATUS_DEBUG "@HUNTER_STATUS_DEBUG@")
+
+if("@MSVC@")
+  hunter_fatal_error("Autotools scheme not supported with Visual Studio")   
+endif()
+
+list(APPEND CMAKE_MODULE_PATH "@HUNTER_SELF@/cmake/find")
+
+set(ODB_ROOT "@ODB_ROOT@")
```

Comment:
This suggestion of yours, together with the comment that you mention that Hunter add itself to `CMAKE_FIND_ROOT` at https://github.com/ruslo/hunter/blob/f711af744ade76be830307656dbded3c4cf244c4/cmake/Hunter#L73 made it not necessary for me to add `ODB_ROOT` manually as I was doing inside the `if(ANDROID) ...` at https://github.com/ruslo/hunter/pull/175/files#diff-1c166191064e9c15bd530aeae37bb9b7R97

Thanks, working perfectly now!